azurerm/resource_arm_storage_blob.go

```diff
@@ -53,6 +54,12 @@ func resourceArmStorageBlob() *schema.Resource {
 				Default:      0,
 				ValidateFunc: validateArmStorageBlobSize,
 			},
+			"content_type": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Default:       "application/octet-stream",
```

Comment:
where does this default come from?

Reply:
The default value comes from `Content-Type` section in https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob#request-headers-all-blob-types.